http/src/main/resources/init-resources/jupyter-docker-compose-gce.yaml

```diff
@@ -2,19 +2,23 @@
 # configuring memory options in container mode. See discussion in:
 # https://docs.docker.com/compose/compose-file/#resources
 # https://github.com/docker/compose/issues/4513
-version: '2'
+version: '2.4'
 services:
   jupyter:
     container_name: "${JUPYTER_SERVER_NAME}"
     image: "${JUPYTER_DOCKER_IMAGE}"
     # Override entrypoint with a placeholder to keep the container running indefinitely.
     # The cluster init script will run some scripts as root and then start pyspark as
     # jupyter-user via docker exec.
-    entrypoint: "tail -f /dev/null"
-    network_mode: host
+    # -F will follow the log when the log is created.
+    entrypoint: "tail -F /home/jupyter/notebooks/jupyter.log"
```

Comment:
Could this be:
```
entrypoint: "tail -F ${NOTEBOOKS_DIR}/jupyter.log"
```
?

That way it would still work on old and new hom directories. (Right now it says "file does not exist" on current runtimes.)

Reply:
oh yea...definitely...thanks for the catch!